Disable Valgrind for LoongArch64

MingcongBai opened the pull request:
Topic Description
-----------------

- groups: drop references to mongodb
- pymongo: drop, orphaned
- mongo-tooling-metrics: drop, orphaned
- mongodb: drop, orphaned
- gdal: drop mongodb dep
- python-3: update to 3.10.13
    - Track patches at AOSC-Tracking/cpython.
    - Lint script in accordance with the Styling Manual.
    - Adapt build scripts for Autobuild4.
- gnome-software: (loongarch64) add fwupd dep
- openmpi: update to 4.1.6
    - Track patches at AOSC-Tracking/ompi.
- qtcreator: update to 12.0.1
    - This update fixes build with Qt 6.6.
- mesa: (loongarch64) disable Valgrind as it is not yet available
- libxkbcommon: (loongarch64) disable Valgrind as it is not yet available
- debug-base: (loongarch64) disable Valgrind as it is not yet available
- python-3: (loongarch64) disable Valgrind as it is not yet available
- gnome-software: (loongarch64) disable Valgrind as it is not yet available
- gnome-builder: (loongarch64) disable Valgrind as it is not yet available
- openmpi: (loongarch64) disable Valgrind as it is not yet available
- qtcreator: (loongarch64) disable Valgrind as it is not yet available
- rustc-demangle: use custom build script

Package(s) Affected
-------------------

- rustc-demangle: 0.1.21-5
- gnome-software: 42.4
- gnome-builder: 42.1-4
- qtcreator: 12.0.1
- debug-base: 2-2
- openmpi: 4.1.6
- python-3: 3.10.13
- libxkbcommon: 1.6.0
- mesa: 1:24.0.0+dxheaders1.611.0
- gdal: 3.7.0-4

Security Update?
----------------

No

Build Order
-----------

```
#buildit rustc-demangle qtcreator openmpi gnome-builder gnome-software python-3 debug-base libxkbcommon mesa gdal
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

MingcongBai committed:
python-3: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
qtcreator: (loongarch64,riscv64) disable Valgrind as it is not yet available

MingcongBai committed:
gnome-builder: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
mesa: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
openmpi: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
mongodb: drop, orphaned

MingcongBai committed:
libxkbcommon: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
gnome-software: (loongarch64) disable Valgrind as it is not yet available

MingcongBai committed:
debug-base: (loongarch64) disable Valgrind as it is not yet available

jiegec commented:
```
[[31mERROR[0m]: [1mAPI removed: user and group commands are removed in Autobuild4[0m
[[96mINFO[0m]:  [1mPlease use systemd-sysusers instead. See sysusers.d(5) for more information.[0m
[[96mINFO[0m]:  [1mYou may use the following auto-generated configuration file as a start:[0m
#Type Name       ID                  GECOS              Home directory Shell
u    mongodb  300:daemon "MongoDB daemon owner" /var/lib/mongodb  /bin/bash
```

MingcongBai committed:
qtcreator: update to 12.0.1

This update fixes build with Qt 6.6.

MingcongBai committed:
openmpi: update to 4.1.6

Track patches at AOSC-Tracking/ompi.

MingcongBai committed:
gnome-software: (loongarch64) add fwupd dep

MingcongBai committed:
python-3: update to 3.10.13

- Track patches at AOSC-Tracking/cpython.
- Lint script in accordance with the Styling Manual.
- Adapt build scripts for Autobuild4.

MingcongBai committed:
mongo-tooling-metrics: drop, orphaned

MingcongBai committed:
pymongo: drop, orphaned

MingcongBai committed:
groups: drop references to mongodb

MingcongBai committed:
gdal: drop mongodb dep

MingcongBai committed:
rustc-demangle: use custom build script

- This package does not make use of the standard cargo install routine.
- Autobuild4 does not seem to handle this package well, with it failing to
  recognise it as a package that uses `cargo build'.

MingcongBai committed:
qtcreator: (loongson3) append -lpthread to fix build
